Ports/.port_include.sh

```diff
@@ -412,6 +412,11 @@ func_defined pre_configure || pre_configure() {
     :
 }
 func_defined configure || configure() {
+    if [[ "$configscript" == "cmake" ]]; then
+        run cmake "${configopts[@]}"
```

Comment:
This looks reasonable, but we should add the `-DCMAKE_TOOLCHAIN_FILE` argument by default, similar to how the `--host` argument is added to `./configure` scripts. Could you also convert at least one port (but probably not *all* of them) to use this pattern as a second commit so we can see what it looks like in use?

Reply:
Done